src/SW_Site.c

```diff
@@ -181,68 +181,74 @@ static Bool SW_check_soil_properties(
         errtype = Str_Dup("gravel content", LogInfo);
 
-    } else if (LE(SW_Site->fractionWeightMatric_sand[layerno], 0.) ||
-               GE(SW_Site->fractionWeightMatric_sand[layerno], 1.)) {
+    } else if (LE(SW_Site->soils.fractionWeightMatric_sand[layerno], 0.) ||
+               GE(SW_Site->soils.fractionWeightMatric_sand[layerno], 1.)) {
         res = swFALSE;
-        fval = SW_Site->fractionWeightMatric_sand[layerno];
+        fval = SW_Site->soils.fractionWeightMatric_sand[layerno];
         errtype = Str_Dup("sand proportion", LogInfo);
 
-    } else if (LE(SW_Site->fractionWeightMatric_clay[layerno], 0.) ||
-               GE(SW_Site->fractionWeightMatric_clay[layerno], 1.)) {
+    } else if (LE(SW_Site->soils.fractionWeightMatric_clay[layerno], 0.) ||
+               GE(SW_Site->soils.fractionWeightMatric_clay[layerno], 1.)) {
         res = swFALSE;
-        fval = SW_Site->fractionWeightMatric_clay[layerno];
+        fval = SW_Site->soils.fractionWeightMatric_clay[layerno];
         errtype = Str_Dup("clay proportion", LogInfo);
 
-    } else if (GE(SW_Site->fractionWeightMatric_sand[layerno] +
-                      SW_Site->fractionWeightMatric_clay[layerno],
+    } else if (GE(SW_Site->soils.fractionWeightMatric_sand[layerno] +
+                      SW_Site->soils.fractionWeightMatric_clay[layerno],
                   1.)) {
         res = swFALSE;
-        fval = SW_Site->fractionWeightMatric_sand[layerno] +
-               SW_Site->fractionWeightMatric_clay[layerno];
+        fval = SW_Site->soils.fractionWeightMatric_sand[layerno] +
+               SW_Site->soils.fractionWeightMatric_clay[layerno];
         errtype = Str_Dup("sand+clay proportion", LogInfo);
 
-    } else if (LT(SW_Site->impermeability[layerno], 0.) ||
-               GT(SW_Site->impermeability[layerno], 1.)) {
+    } else if (LT(SW_Site->soils.impermeability[layerno], 0.) ||
+               GT(SW_Site->soils.impermeability[layerno], 1.)) {
         res = swFALSE;
-        fval = SW_Site->impermeability[layerno];
+        fval = SW_Site->soils.impermeability[layerno];
         errtype = Str_Dup("impermeability", LogInfo);
 
-    } else if (LT(SW_Site->fractionWeight_om[layerno], 0.) ||
-               GT(SW_Site->fractionWeight_om[layerno], 1.)) {
+    } else if (LT(SW_Site->soils.fractionWeight_om[layerno], 0.) ||
+               GT(SW_Site->soils.fractionWeight_om[layerno], 1.)) {
         res = swFALSE;
-        fval = SW_Site->fractionWeight_om[layerno];
+        fval = SW_Site->soils.fractionWeight_om[layerno];
         errtype = Str_Dup("organic matter content", LogInfo);
 
-    } else if (LT(SW_Site->evap_coeff[layerno], 0.) ||
-               GT(SW_Site->evap_coeff[layerno], 1.)) {
+    } else if (LT(SW_Site->soils.evap_coeff[layerno], 0.) ||
+               GT(SW_Site->soils.evap_coeff[layerno], 1.)) {
         res = swFALSE;
-        fval = SW_Site->evap_coeff[layerno];
+        fval = SW_Site->soils.fractionWeight_om[layerno];

```

Comment:
SW_check_soil_properties(): this looks like an error during merge conflict resolution
```
    } else if (LT(SW_Site->soils.evap_coeff[layerno], 0.) ||
               GT(SW_Site->soils.evap_coeff[layerno], 1.)) {
        res = swFALSE;
        fval = SW_Site->soils.fractionWeight_om[layerno];
        errtype = Str_Dup("organic matter content", LogInfo);
```
and should be removed: fractionWeight_om is checked on lines 227-231 and evap_coeff is checked on lines 239-243 (see also https://github.com/DrylandEcology/SOILWAT2/commit/8f895992f208793d33d92746c3eca9382febec9f#diff-f9371e34e93e7bab2198e93bdaccdbd4186f5127f01d95aa44da7ea9d26db0e5)